### PR TITLE
Add Contact Form

### PR DESCRIPTION
**Pull request by Alfred-KInyua**
My Name field accepts up to 30 chars.
My Email field accepts only emails.
My Message field accepts up to 500 chars.
All my fields have placeholder text.
All my fields have the correct type.
A user cannot submit the form if one or more of the fields is empty.

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -47,149 +46,179 @@ <h2 class="desktop">Hi there. I'm Alfred</h2>
...
-                <input type="text" id="fname" name="fname" class="input_one" placeholder="Full name"><br>
-                <input type="email" id="lname" name="email" class="input_one" placeholder="Email address"><br><br>
-                <textarea id="talk" name="talk" class="message_box" placeholder="Enter text here here"></textarea>
-                <input type="submit" class="button" value="Get in touch">
+            <form  action="https://formspree.io/f/mknypyqz" method="POST" class="forms">
+                <input type="text" id="fname" name="name" class="input_one" placeholder="First name" required maxlength="30">
+                <input type="text" id="lname" name="name" class="input_one last-input desktop" placeholder="Last name" required maxlength="30">
+                <input type="email" name="_replyto" class="input_two" placeholder="Email address" required><br><br>
+                <textarea id="message" name="message" class="message_box" placeholder="Enter text here here" required maxlength="500"></textarea>
+                <input type="submit" class="button small-button" value="Get in touch">
```
Comment (OLIPLICHE) [inline index.html:211]: - [ ] Please your submit button is not working. Kindly fix that and make sure the form is submitted successfully.

**Review by OLIPLICHE — CHANGES_REQUESTED** [review]
Hi @Alfred-KInyua , 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

## Highlights
- Linter is green (the last commit have a green check mark (✅)) 👏
- README file is descriptive 👏
- Commit messages are meaningful 👏 👏
- Github flow is correctly used (Pull Request is made from the feature branch into the main branch) 👏
- Descriptive pull request👏

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

You can also consider:
- x
- y
- z

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by vechicin — CHANGES_REQUESTED** [review]
# Hi @Alfred-KInyua  ✋, 

Great work on making the changes requested by a previous reviewer 👏 
You've done well implementing some of the requested changes, but there are still some which aren't addressed yet. 🚨 


 ## Highlights ⚡ 

- [x] Good job implementing the changes requested by the previous reviewer

 ## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions 📂 

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by ShadyShawkat — APPROVED** [review]
## Hi @Alfred-KInyua, 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

Image

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**